src/checkout-node-query.js

```diff
@@ -0,0 +1,41 @@
+import createGid from './create-gid';
+import shippingRateQuery from './shipping-rate-query';
+import lineItemConnectionQuery from './line-item-connection-query';
+import mailingAddressQuery from './mailing-address-query';
+import customAttributeQuery from './custom-attribute-query';
+import orderQuery from './order-query';
+import addFields from './add-fields';
+
+const defaultFields = [
+  'id',
+  'ready',
+  ['lineItems', lineItemConnectionQuery()],
+  ['shippingAddress', mailingAddressQuery()],
+  ['shippingLine', shippingRateQuery()],
+  'requiresShipping',
+  ['customAttributes', customAttributeQuery()],
+  'note',
+  'paymentDue',
+  'webUrl',
+  ['order', orderQuery()],
+  'orderStatusUrl',
+  'taxExempt',
+  'taxesIncluded',
+  'currencyCode',
+  'totalTax',
+  'subtotalPrice',
+  'totalPrice',
+  'completedAt',
+  'createdAt',
+  'updatedAt'
+];
```

Comment:
We probably don't need `ready` if we're not using any of the `checkoutComplete` mutations. I'm not sure if we need `paymentDue` if we're not exposing discount/gift card mutations, `order` or `orderStatusUrl`. Otherwise I think the rest could be used.